src/AspNet.Security.OAuth.Introspection/Events/AuthenticationFailedContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
```

Comment:
I'd be picky if I asked you to use K&R everywhere (I'll fix that myself) :smile: